modules/twinklewarn.js

```diff
@@ -537,74 +649,126 @@ Twinkle.warn.messages = {
 				label: "Ownership of articles",
 				summary: "Warning: Ownership of articles"
 			},
+			"uw-taxonomy3": {
+				label: "Disrupting the taxonomy templates",
+				summary: "Warning: Disrupting the taxonomy templates"
+			},
 			"uw-tdel3": {
 				label: "Removal of maintenance templates",
 				summary: "Warning: Removal of maintenance templates"
 			},
-			"uw-unsourced3": {
-				label: "Addition of unsourced or improperly cited material",
-				summary: "Warning: Addition of unsourced or improperly cited material"
+			"uw-test3": {
+				label: "Editing tests",
+				summary: "Warning: Editing tests"
+			},
+			"uw-upload3": {
+				label: "Uploading unencyclopedic images",
+				summary: "Warning: Uploading unencyclopedic images"
+			}
+		},
+		"Behavior towards other editors": {
+			"uw-agf3": {
+				label: "Not assuming good faith",
+				summary: "Warning: Not assuming good faith"
+			},
+			"uw-harass3": {
+				label: "Harassment of other users",
+				summary: "Warning: Harassment of other users"
+			},
+			"uw-npa3": {
+				label: "Personal attack directed at a specific editor",
+				summary: "Warning: Personal attack directed at a specific editor"
```

Comment:
Almost sounds inviting: Next time, please attack multiple editors instead. 😉 So this, too, is not useful in the edit summary but rather only in the Twinkle interface.

Reply:
Feel free to merge unmodified, I'll fix this shortly afterwards. It won't hurt and seems to be the easiest and fastest solution. 😄